Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteAboutController.java

```diff
@@ -0,0 +1,31 @@
+package edu.asu.spring.quadriga.web.publicwebsite;
+
+import java.security.Principal;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.spring.quadriga.domain.workbench.IProject;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.workbench.IRetrieveProjectManager;
+
+@Controller
+public class WebsiteAboutController {
```

Comment:
Also, please adjust your eclipse settings to use 4 spaces:
https://diging.atlassian.net/wiki/pages/viewpage.action?pageId=8781898